packages/blade/src/components/IconButton/__tests__/IconButton.native.test.tsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/no-empty-function */
+
+import { fireEvent } from '@testing-library/react-native';
+
+import IconButton from '../';
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.native';
+import { CloseIcon } from '~components/Icons';
+
+describe('<IconButton />', () => {
+  it('should render', () => {
+    const noop = () => {};
```

Comment:
Intentionally passing explicit function because I don't want to test the mock function in this block. Added a separate test case for mock function callback.